octopusdeploy/schema_space.go

```diff
@@ -73,18 +73,19 @@ func getSpaceDataSchema() map[string]*schema.Schema {
 
 	return map[string]*schema.Schema{
 		"id":           getDataSchemaID(),
-		"ids":          getQueryIDs(),
-		"name":         getQueryName(),
-		"partial_name": getQueryPartialName(),
-		"skip":         getQuerySkip(),
+		"name":         withConflictsWith(getQueryName(), []string{"id", "ids", "partial_name", "skip", "take"}),
+		"ids":          withConflictsWith(getQueryIDs(), []string{"name"}),
+		"partial_name": withConflictsWith(getQueryPartialName(), []string{"name"}),
```

Comment:
I'm confused about the `withConflictsWith`, so this is supposed to be a xor boolean?
Becasue eg, if the user supplies both `partialName` and `ids`, what happens then?